runtime/src/bank.rs

```diff
@@ -4048,11 +4048,29 @@ impl Bank {
         {
             let block_cost_limit = simd_0286_block_limits();
             let mut cost_tracker = self.write_cost_tracker().unwrap();
-            let account_cost_limit = cost_tracker.get_account_limit();
+            let account_cost_limit = if self
+                .feature_set
+                .is_active(&feature_set::raise_account_cu_limit::id())
+            {
+                (block_cost_limit as f64 * 0.4) as u64
```

Comment:
there could be rounding if we use f64.

can we do 40/100 or something. as long as CU limits are a multiple of 100 there's no rounding here and I don't need to worry about compilers doing something really dumb with flops.